Readme.md

```diff
@@ -45,41 +45,50 @@ Prior to using Explorer for Endevor, ensure that you meet the following prerequi
 
 Explorer for Endevor is included with Eclipse Che version 7.6.0 and above. Check [here](https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che/) for more information.
 
-
 ## **User Guide**
 
 Create a Host in Explorer for Endevor to enable it to access the repository you want to explore.
 
 ### **Create a Profile**
 
-Create a Profile in Explorer for Endevor to specify the data source that you want to work with. This allows you to view and explore the selected Endevor repositories.
+Create a profile in Explorer for Endevor, then specify the data source that you want to work with. This allows you to view and explore the selected Endevor repositories.
```

Comment:
Data source should be "configuration" throughout. The Endevor team is going to update their documentation accordingly.

Reply:
Done